gpkit/constraints/model.py

```diff
@@ -108,3 +108,51 @@ def subconstr_latex(self, excluded=None):
         "The collapsed appearance of a ConstraintBase"
         if self.name:
             return "%s_{%s}" % (self.name, self.num)
+
+    def debug(self):
+        "Attempts to diagnose infeasible models."
+        from .relax import RelaxConstants, RelaxConstraints
+        from .bounded import Bounded
+
```

Comment:
check solver status here? Mosek, at least, will give a `PRIM_INFEAS_CER` for many primal infeasible models, in which case there's no sense in doing the Bounded formulation...